_data/committers.yml

```
@@ -296,6 +296,10 @@
   role: Committer
   alias: assignuser
   affiliation: Voltron Data
+- name: James Duong
+  role: Committer
+  alias: jduo
```

Is your Apache ID is `judong` not `jduo`?

https://people.apache.org/phonebook.html?uid=jduong

```suggestion
  alias: jduong
```

---

Oh that's correct. My apache is is jduong and my GitHub user is jduo. Thanks